pkg/controller/container-runtime-config/helpers.go

```diff
@@ -94,7 +95,11 @@ func createNewIgnition(configs map[string][]byte) igntypes.Config {
 }
 
 func findStorageConfig(mc *mcfgv1.MachineConfig) (*igntypes.File, error) {
```

Comment:
These four copies of essentially the same code keep getting larger and more unwieldy (and _only_ two contain the `c := c` copy statement).  Consider replacing it with a `findConfigFile(mc, path)`?

Reply:
That is an optimization unrelated to the RawExt conversion and can be done separately if needed.